sys/vm/vm_unix.c

```diff
@@ -177,12 +177,12 @@ kern_break(struct thread *td, uintptr_t *addr)
 			PROC_UNLOCK(td->td_proc);
 		}
 #endif
-		prot = VM_PROT_RW;
+		prot = VM_PROT_RW_CAP;
```

Comment:
Yeah, I just meant to exclude riscv and arm64 where this is mostly dead code (actually should be deader on mips since 32-bit support wouldn't include break there.)